doc/core.md

```diff
@@ -142,7 +147,12 @@ Postcomputations cannot access stream values, but we can assume that all reducti
 What about "numflips" or a sum over all time?
 Since it's simply a reduce over all previous data, and the previous data isn't going to change, we should be able to just use the previous reduce value etc.
 
-So in summary, each fold reduction should output its current value as the lookbehind, and each latest reduction should output the values it used.
+There are three types of reductions: 
+
+- latest N: the used facts are stored
```

Comment:
Ah no, I hadn't thought of that really - the n is just the number of elements here.
So for implementing something like "first" I was thinking we could use a fold over all time:
```fold(\acc elem -> Just elem) Nothing```
and then we'd store the value of that reduce ie (Just Foo) in the history.

For latest, I'm only allowing conditions like filters and windows right now. I had "take" like firstN before, but it actually complicates lookbehind a bit, and I figured it could be implemented with a fold.